src/kfactory/kcell.py

```diff
@@ -163,26 +163,30 @@ class KCellSettings(BaseModel, extra="allow", validate_assignment=True, frozen=T
     @model_validator(mode="before")
     def restrict_types(
         cls, data: dict[str, Any]
-    ) -> dict[str, int | float | SerializableShape | str]:
+    ) -> dict[str, int | float | SerializableShape | str | tuple[float | int, ...]]:
         for name, value in data.items():
-            if not isinstance(value, str | int | float | SerializableShape):
+            if not isinstance(value, str | int | float | SerializableShape | tuple):
                 data[name] = str(value)
         return data
 
     def __getitem__(self, key: str) -> Any:
         return getattr(self, key)
 
+    def get(self, __key: str, default: Any | None = None) -> Any:
+        return getattr(self, __key) if hasattr(self, __key) else default
```

Comment:
Fine by me, but `getattr` supports a default value by design already. Also, `Any | None`is redundant as `Any` includes `None` afaik

Reply:
I fixed it :) thank you